app/src/main/java/com/abhijitvalluri/android/fitnotifications/AppSettingsActivity.java

```diff
@@ -116,7 +118,8 @@ public void onClick(View v) {
                                                                            mStartTimeMinute,
                                                                            mStopTimeHour,
                                                                            mStopTimeMinute,
-                                                                           R.string.start_time_heading,
+                                                                           stopTimeFormatted,
```

Comment:
Remove this because this causes a logic bug. After the user updates the start time or stop time, this becomes outdated. 

So if say, initially, the schedule is 6:00 AM to 11:00 AM. Then, the user updates the start time to 7:00 AM. Then, if the user now tries to set the stop time as 6:30 AM, it will correctly deny the user the option of saving it as it is less than 7:00 AM. __BUT__, it will incorrectly show in the toast that it should be greater than the old start time (6:00 AM). It seems overkill to show the exact stop time in the toast. 

You can fix it by updating the OnClickListener each time, but it is easier to just remove it.

Also, please thoroughly test your changes before submitting them to avoid such errors.